google/api_core/rest_streaming.py

```diff
@@ -107,7 +114,10 @@ def __next__(self):
 
     def _grab(self):
         # Add extra quotes to make json.loads happy.
-        return self._response_message_cls.from_json(self._ready_objs.popleft())
+        if issubclass(self._response_message_cls, proto.Message):
+            return self._response_message_cls.from_json(self._ready_objs.popleft())
+        else:
```

Comment:
nit: for safety and future-proofing, maybe this should be an `elif` and we check for `google.protobuf.Message`
(I realize we don't expect to ever have the second check fail if we get here....but "we don't expect" == "famous last words")

(not a blocker)

Reply:
Fixed in https://github.com/googleapis/python-api-core/pull/604/commits/9f5b22f370e6546c9274eb25fcf1088fc07c843f